docs/guides/server/hostname.adoc

```diff
@@ -107,21 +461,24 @@ In this example, the server is running without a proxy and exposed using a URL u
 It is highly recommended using a TLS termination proxy in front of the server for security and availability reasons. For more details,
 see <@links.server id="reverseproxy"/>.
 
-=== Forcing backend endpoints to use the same URL the server is exposed
+==== Forcing backend endpoints to use the same URL the server is exposed
 
 In this example, backend endpoints are exposed using the same URL used by the server so that clients always fetch the same URL
 regardless of the origin of the request.
 
 .Keycloak configuration:
 <@kc.start parameters="--hostname=mykeycloak --hostname-strict-backchannel=true"/>
 
-=== Exposing the server using a port other than the default ports
+==== Exposing the server using a port other than the default ports
 
 In this example, the server is accessible using a port other than the default ports.
 
 .Keycloak configuration:
 <@kc.start parameters="--hostname-url=https://mykeycloak:8989"/>
 
+
+
+
```

Comment:
I would suggest to remove the redundant new line characters.

Reply:
Removed.